source/shared/net/guides/baker-pool.rst

```diff
@@ -0,0 +1,43 @@
+.. _baker-pool:
+
+================
+Baker management
+================
+
+A :ref:`baker <baker-concept>` is a node that participates in the network by baking (creating) new blocks that are added to the chain. There are some recommended best practices around baking, particularly when a baker runs a baker pool. A baker can choose to open a baker pool to which others can :ref:`delegate<delegation-concept>` some of their stake.
+
+Best practices for baking
+=========================
+
+As a baker you should manage your baker node responsibily for the benefit of all users of the blockchain. The recommendations for bakers are:
+
+- Make sure your node is running 24/7/365 in a reliable place. 
+- Protect your baker keys so that they are not lost or compromised.
+- Check regularly to make sure that you are not close to the bounding cap for max capital for a baker. If you're getting close to the max capital, you can split and make another baker to divide the capital.
+
+Recommendations for baker pool owners
+-------------------------------------
+
+If you are running a baker pool, you have a responsibility to manage your pool responsibly to maximize returns, both for yourself and those who have delegated stake to your pool.
+
+To help potential delegators with their choice, it is a good idea to create a site with information and add this URL to your baker pool configuration. This information is published on :ref:`CCD Scan<ccd-scan>` so that potential delegators can read about the pool and understand your goals. Having a URL on your baker pool configuration is not required.
+
+You should regularly check to ensure that you aren't close to the leverage cap (too many delegators). If so, you can close the pool to new delegators.
```

Comment:
I think remove this. Delegators are automatically prevented from delegating over the cap, so I closing the pool is unnecessary.
```suggestion
```